package.json

```diff
@@ -125,6 +125,7 @@
     "exit_on_eof": "^1.0.4",
     "imports-loader": "^0.7.1",
     "lint-staged": "^7.3.0",
+    "medium-editor": "^5.23.3",
```

Comment:
`medium-editor` dependency is already present in the `master`'s. Don't forget to merge master branch to the dev branch